src/charm.py

```diff
@@ -181,6 +223,9 @@ def _generate_cu_config(self) -> str:
         if not self._n2_requirer.amf_ip_address:
             logger.warning("AMF IP address not available")
             return ""
+        if (n3_interface_name := self._get_n3_interface_from_config()) is None:
+            n3_interface_name = N3_INTERFACE_NAME
+
```

Comment:
If we had a default value for the N3 interface, this could probably be removed.

Reply:
Removed.